javascripts/player.js

```diff
@@ -1,14 +1,11 @@
 portals = {
     IPFS: [
-        "https://video.dtube.top/ipfs/",
         "https://video.oneloveipfs.com/ipfs/",
-        "https://ipfs.busy.org/ipfs/",
-        "https://ipfsgateway.makingblocks.xyz/ipfs/",
-        "https://ipfs.io/ipfs/",
         "https://ipfs.infura.io/ipfs/",
         "https://gateway.temporal.cloud/ipfs/",
         "https://gateway.pinata.cloud/ipfs/",
-        "https://ipfs.eternum.io/ipfs/"
+        "https://ipfs.eternum.io/ipfs/",
+        "https://ipfs.io/ipfs/"
```

Comment:
```suggestion
        "https://gateway.ipfs.io/ipfs/"
```

For clarity, I'd encourage you to hit `gateway.ipfs.io`, instead of just `ipfs.io`. Same functionality otherwise.